MTM-62347: Fixed NullPointerException when user was queries Measureme… (#2888)
SoftwareAG/c8y-docs: content/change-logs/platform-services/cumulocity-2025.44.0-fix-NPE-when-query-measurement-API-using-valueSeriesFragment-with-inventory-roles-permissions.md

---

> date: '2025-02-04'

Reviewer: ```suggestion
date: 
```

Reviewer: The date will be set automatically on deployment

> title: Fixed NullPointerException when user was queries Measurement API 

Reviewer: ```suggestion
title: Queries to the Measurement API with the filter valueSeriesFragment no longer result in a NullPointerException
```

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-JlFdtOPva
    label: Rest API
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: MTM-62347
version: 2025.44.0
---

> When user with inventory roles permission for Api: `Measurement`, fragment: `*` was triggering an query to measurement api,

Reviewer: ```suggestion
When users with inventory roles permission for the API `Measurement` and the fragment `*` were triggering a query to the Measurement API
```

> using filter `valueSeriesFragment` it resulted in NullPointerException and 500 error code.

Reviewer: ```suggestion
using the filter `valueSeriesFragment`, this resulted in a NullPointerException and error code 500.
```

> The issue was now solved and filter can be used.

Reviewer: ```suggestion
The issue has been fixed and the filter can be used properly.
```
